specification/vi/resource-manager/Microsoft.VideoIndexer/preview/2022-04-13-preview/vi.json

```diff
@@ -0,0 +1,1279 @@
+          "example": "/subscriptions/xxx/resourceGroups/contoso-videoAnalyzer-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/contoso-videoAnalyzer-mi"
+        }
+      }
+    },
+    "AccessToken": {
+      "description": "Azure Video Analyzer for Media access token.",
+      "type": "object",
+      "properties": {
+        "accessToken": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The access token.",
+          "x-ms-secret": true,
+          "example": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJPbmxpbmUgSldUIEJ1aWxkZXIiLCJpYXQiOjE2MjI0NDc3MzMsImV4cCI6MTY1Mzk4MzczMywiYXVkIjoid3d3LmV4YW1wbGUuY29tIiwic3ViIjoianJvY2tldEBleGFtcGxlLmNvbSIsIkdpdmVuTmFtZSI6IkpvaG5ueSIsIlN1cm5hbWUiOiJSb2NrZXQiLCJFbWFpbCI6Impyb2NrZXRAZXhhbXBsZS5jb20iLCJSb2xlIjpbIk1hbmFnZXIiLCJQcm9qZWN0IEFkbWluaXN0cmF0b3IiXX0"
+        }
+      }
+    },
+    "UserClassicAccountList": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClassicAccountSlim"
+          },
+          "description": "List of classic account names and their location."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of operation list results if there are any.",
+          "example": "https://management.azure.com/{accountsOperation}?api-version={version}&%24skiptoken={token}"
+        }
+      },
+      "description": "The list of user classic accounts.",
+      "type": "object"
+    },
+    "ClassicAccountSlim": {
+      "description": "An Azure Video Analyzer for Media classic account.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The account's name",
+          "type": "string",
+          "readOnly": true,
+          "example": "contoso-videoanalyzer"
+        },
+        "location": {
+          "description": "The account's location",
+          "type": "string",
+          "readOnly": true,
+          "example": "westus"
+        },
+        "id": {
+          "description": "The account's id",
+          "type": "string",
+          "readOnly": true,
+          "example": "462af7c5-d1f6-4b91-86e3-8bc5e8a61574"
+        }
+      }
+    },
+    "ClassicAccount": {
+      "description": "An Azure Video Analyzer for Media classic account.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The account's name",
+          "type": "string",
+          "readOnly": true,
+          "example": "contoso-videoanalyzer"
+        },
+        "location": {
+          "description": "The account's location",
+          "type": "string",
+          "example": "westus"
+        },
+        "id": {
+          "description": "The account's id",
+          "type": "string",
+          "readOnly": true,
+          "example": "462af7c5-d1f6-4b91-86e3-8bc5e8a61574"
```

Comment:
>462af7c5-d1f6-4b91-86e3-8bc5e8a61574" [](http://example.com/codeflow?start=22&length=37)

Wondering why it is not a fully qualify resource ID for ARM resources?

[https://armwiki.azurewebsites.net/introduction/concepts/resourceids.html](https://armwiki.azurewebsites.net/introduction/concepts/resourceids.html)

Reply:
@j5lim 
Classic Accounts are not Arm accounts (not an arm resource)